squad/api/rest.py

```diff
@@ -622,18 +622,20 @@ def log_file(self, request, pk=None):
     @detail_route(methods=['get'], suffix='tests')
     def tests(self, request, pk=None):
         testrun = self.get_object()
-        tests = testrun.tests.prefetch_related('suite')
-        page = self.paginate_queryset(tests)
+        tests = testrun.tests.prefetch_related('suite').order_by('id')
+        paginator = PageNumberPagination()
```

Comment:
Maybe CursorPagination would be better here? There can be a lot of tests and count(*) required for PageNumberPagination is very expensive.

Reply:
I take this back. Just pulled the code and tested. It works. The downside is that the API doesn't provide the number of pages. But I can live with that (at least for now).